sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/aio/_phone_numbers_client_async.py

```diff
@@ -0,0 +1,236 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.paging import ItemPaged
+from azure.core.polling import LROPoller
+from .._generated.aio._phone_numbers_client import PhoneNumbersClient as PhoneNumbersClientGen
```

Comment:
Is 'import {A class} as {A Class}Gen' a common practice? why we need to do this? 
I don't see next line is doing the same

Reply:
It's also done in the Identity client and it was also done in the previous implementation of this PNM SDK, so I just left it as it was